test/functional/test_framework/messages.py

```diff
@@ -142,6 +144,52 @@ def uint256_from_compact(c):
     return v
 
 
+def deser_fixed_bytes(f, size):
+    r = []
+    for i in range(size):
+        r.append(struct.unpack("B", f.read(1))[0])
+    return r
+
+def ser_fixed_bytes(u, size):
+    rs = b""
+    for i in range(size):
+        rs += struct.pack("B", u[i])
+        #rs += struct.pack("B", u & 0xFF)
+        #u >>= 8
+    return rs
+    
+
+def deser_pubkey(f):
+    r = 0
+    for i in range(33):
+        t = struct.unpack("B", f.read(1))[0]
+        r += t << (i * 8)
+    return r
+
+def ser_pubkey(u):
+    rs = b""
+    for _ in range(33):
+        rs += struct.pack("B", u & 0xFF)
+        u >>= 8
+    return rs
+    
+
+def deser_signature(f):
```

Comment:
Seems identical to the pubkey versions apart from the length, perhaps consider factoring into a common function?

Reply:
Sorry for dropping mid-conversation @hectorchu  but I wanted to see if we could chat via Telegram or some other channel?  
I'm the team lead for Litewallet, the mobile wallet from Litecoin Foundation and we have an initiative to integrate MWEB.  Will you let me know if this is possible? Thanks!

Reply:
Hi Kerry, yes you can contact me on telegram. Look for me in the MWEB dev group.